tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2249,18 +2266,23 @@ public function testCheckTaxAmount(string $thousandSeparator): void {
     $this->createFinancialTypeWithSalesTax();
     $financialTypeId = $this->ids['FinancialType']['taxable'];
 
-    $params = array_merge($this->_params, ['contribution_status_id' => 2, 'financial_type_id' => $financialTypeId]);
-    $contribution = $this->callAPISuccess('contribution', 'create', $params);
-    $contribution1 = $this->callAPISuccess('contribution', 'get', ['id' => $contribution['id'], 'return' => 'tax_amount', 'sequential' => 1]);
-    $this->callAPISuccess('contribution', 'completetransaction', [
-      'id' => $contribution['id'],
+    $params = array_merge($this->_params, ['contribution_status_id' => 2, 'sequential' => 1, 'financial_type_id' => $financialTypeId]);
```

Comment:
@eileenmcnaughton can you please change the 2 to 'Completed' !!

Reply:
I just moved it down - see it's a few lines above - - but it should be Pending if I'm changing it

Reply:
oops I mean changing 2 to 'Pending'.

Reply:
OK - updated - I actually flicked it over to Order.create which no longer requires contribution_status_id (forces it to pending)